README.md

```diff
@@ -54,50 +54,47 @@ Project is designed to utilize django's powerful features.
-2. Once you have mysql downloaded, create a new database inside your mysql.
-    a. Login to mysql by using  `mysql --u root --p` and then type in the password you saved previously when prompted
-        i. Do note that root is the default username so use whatever username you used
-    b. You should be prompted into they mysql shell, and then create a database by using `CREATE DATABASE mesh;`
-    c. Confirm your database is created with `SHOW DATABASES;`
-3. make sure your pip environment is opened (with `pipenv shell`)
-4. Type in `python manage.py makemigrations`
-    a. You will most likely be given warnings. Warnings are fine errors aren’t. Google your errors.
-5. Type in `python manage.py migrate`
-    a. Similar to the previous step warnings are fine, errors aren’t.
-6. Make sure your frontend is also set up (with npm run build) and then you are good to go. Run `python manage.py runserver` and you should be good to go.
+Check node version with `node -v` -> This should show v20.2.0 in the console.
+Check npm version with `npm -v` -> This should show 9.6.6 in the console.
+
+1. Clone and change directories to the repository in the directory (ex. Desktop) you would like to work in. `git clone [repo link]`.
+2. Development is done on the `dev_new` branch. Ensure you are working either on `dev_new` or a branch that has branched off of `dev_new`. To switch to a branch, use `git checkout [branch_name]`.
+3. Install pipenv with pip using `pip install pipenv`.
+   - If pip is not recognized, you can do `py -m ensurepip` to check that pip is installed, then restart your Terminal.
+4. **IMPORTANT** Get the .env file from another contributor and store it on your root directory '/SITE' if you haven't already before running this command.
+5. Initialize the virtual environment and install virtual environment dependencies using `pipenv install`.
+6. Activate the virtual environment using `pipenv shell`.
+7. Go to `Site/meshapp/` and run an `npm install`. You can change your directory to this folder using `cd meshapp`.
+8. Build the frontend by running `npm run build`.
+9. Return to the root directory and start the backend server / run the application with `python manage.py runserver`.
+    - `cd ..` (Windows) or `cd -` (MacOS) to go back a directory.
+
+**MYSQL SETUP** (If you are on a Mac, consider switching to Windows if you have one)
+1. Download MySQL over [here](https://dev.mysql.com/downloads/installer/). Download the larger file.
+2. Do the setup and do the full download (spam next until you see the password portion).
+3. Make a password for the root account or a custom username.
+   - Make sure you remember your passwords and username (username is by default root).
+4. Write the username and password in your .env file.
+5. Spam next until you finish and then you’re done.
+
+Next we're onto Django.
```

Comment:
This isn't Django not sure why it was originally stated as such. The steps below are still MySQL set up just integrating it to the environment. The first few steps above are related to Django. ex. pipenv install should install django